### DISCUSSION TITLE
Renaming Vector#is_in method to Vector#crossed

### DISCUSSION THREAD

**Discussion by AlainPilon** [roboflow/supervision#401] [Contribution]
Regarding Vector#is_in

https://github.com/roboflow/supervision/blob/ddc036ea3a861f7f771a5694cf3bd98c715374af/supervision/geometry/core.py#L45

```
    def is_in(self, point: Point) -> bool:
        v1 = Vector(self.start, self.end)
        v2 = Vector(self.start, point)
        cross_product = (v1.end.x - v1.start.x) * (v2.end.y - v2.start.y) - (
            v1.end.y - v1.start.y
        ) * (v2.end.x - v2.start.x)
        return cross_product < 0
```

Should'nt it be named `crossed` instead? The current name would work if it was a polygon, as Points are either IN or OUT, but for a Vector it is not applicable.

**Comment by SkalskiP** [comment]
That name is unfortunate. The code checks if the point is to the left or the right of the vector.

**Comment by AlainPilon** [comment]
Yes, initial idea was to rename it `is_on_the_left` (or is that right?) but given the context it is used it, I found `crossed` to be more expressive.

**Reply by SkalskiP** [reply]
We need to make that name more descriptive. We could also provide some docstring to give more clarity.

**Reply by AlainPilon** [reply]
is_left/is_right isnt good either because it depends if the vector was drawn from top to bottom or the reverse. 

Maybe `starboard_side` and `port_side` but it feels too wordy. This is why I liked `crossed` and provide a doc string to explain the returned value based on the direction of the vector.

**Reply by SkalskiP** [reply]
> is_left/is_right isnt good either because it depends if the vector was drawn from top to bottom or the reverse.

I assumed that we use start and end to define vector direction.
